pkg/config/constants/constants.go

```diff
@@ -115,6 +115,9 @@ const (
 
 	TestVMLabel = "istio.io/test-vm"
 
+	SingleStack      = "SingleStack"
```

Comment:
this doesn't belong in Istio constants, it's a a K8S property, and we only use it in test code when we are templating K8S services - we don't need to mention or create references to it outside of test code.

Furthermore, it's already a K8S constant - https://github.com/kubernetes/api/blob/master/core/v1/types.go#L5193 - if we want a constant to use in test code, we should just use those.

Reply:
Sure, let me update the PR to use K8s constants directly.

Reply:
Ty!